### ISSUE TITLE
continue pretrain

### ISSUE THREAD

**Issue by guozhiyao** [JiuhaiChen/BLIP3o#17]
Hello, is the BLIP3o/BLIP3o-Model on Hugging Face based on Qwen-7B? Can we continue pre-training based on this model using existing code?

**Comment by JiuhaiChen** [comment]
Hi, yeah, the current checkpoint is based on Qwen-VL-2.5-7b, you can continue pertain on it. You only need to change the model name and customize your own dataset. Let me know if you have any questions!

**Comment by guozhiyao** [comment]
@JiuhaiChen I continued to train your model using our own data, but found that the results dropped significantly.
I'm not sure if it's a data problem or because you released the model after SFT. Do you have plans to upload the pretrain model? I want to compare the results of the pretrain and SFT models.

<img width="363" alt="Image" src="https://github.com/user-attachments/assets/a53849ec-f4d3-4309-8f9c-7ee07e7f5866" />
train loss

prompt：a cute cate.
release model

<img width="822" alt="Image" src="https://github.com/user-attachments/assets/f45cdffb-dd88-42fd-b9cb-093e33ac210c" />

continue pretrain model

<img width="819" alt="Image" src="https://github.com/user-attachments/assets/d1171482-86ce-4d2a-bbd8-b05478f8e416" />

**Comment by JiuhaiChen** [comment]
Hi! In our experiments, the training loss stayed flat and didn’t correlate with image quality, you can see the pretraining loss curve here:
https://wandb.ai/jchen169/huggingface/reports/Qwen-VL-3B-Diffusion-transformer-1B--VmlldzoxMjg1NDA1MQ

Could you tell me more about your dataset? And how many training steps you are running? 

As for our pretrain checkpoint, we’ll need to go through an internal review before it can be shared.

**Comment by guozhiyao** [comment]
> 你好！在我们的实验中，训练损失保持平稳，并且与图像质量无关。你可以在这里查看训练前的损失曲线： https://wandb.ai/jchen169/huggingface/reports/Qwen-VL-3B-Diffusion-transformer-1B--VmlldzoxMjg1NDA1MQ
> 
> 你能告诉我更多关于你的数据集的信息吗？你进行了多少个训练步骤？
> 
> 至于我们的训练前检查点，我们需要经过内部审查才能分享。

![Image](https://github.com/user-attachments/assets/611434af-5067-4e23-a1e0-94ef5f1c4a46)

@JiuhaiChen The training data is a large-scale mixed Chinese and English t2i data collected by us. The above is the result of your model training 15k on our data, with the loss curve attached above.

I guess it’s because the data quality of SFT is higher, so the effect after SFT is much better. And because the SFT data is distilled from 4o, the data style is more uniform. I found that the styles of the 4 pictures generated by your model are very uniform, unlike the 4 pictures generated after our data fine-tuning, which vary greatly.

**Comment by JiuhaiChen** [comment]
Yeah, you are right. We have similar observation, the model is less diverse after SFT.
